test/suite_implementation.js

```diff
@@ -12,6 +17,7 @@ module.exports = function(style, options, _callback) {
         }
     }
 
+    window.XMLHttpRequest = XMLHttpRequest;
```

Comment:
Is there a reason the default jsdom XHR isn't suitable? Can we document it with a comment?